platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml

```diff
@@ -27,5 +27,5 @@
 - 
   Environment variable:            '`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES`'
   Description: > 
-    Enable the [stepped launch form](https://docs.seqera.io/platform/24.1.0/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
+    Enable the [stepped launch form](https://docs.seqera.io/platform/latest/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
```

Comment:
I find "Set empty" a little confusing. Does this mean to set `TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=` to empty?

Reply:
Literally an empty value, but must be set to enable in all workspaces. If you just leave the env var out altogether, that's a `null` value so then the launch form is _not_ enabled.

Reply:
Ahh, that makes sense.